jenkins/slave-base/Dockerfile

```diff
@@ -86,6 +69,8 @@ RUN if [ -z $SNYK_DISTRIBUTION_URL ] ; then echo 'Skipping snyk installation!' ;
 
 # Set java proxy var.
 COPY set_java_proxy.sh /tmp/set_java_proxy.sh
+RUN cp /usr/local/bin/run-jnlp-client /usr/local/bin/ocp-run-jnlp-client
+COPY ods-run-jnlp-client /usr/local/bin/run-jnlp-client
```

Comment:
Should we ensure here that `/usr/local/bin/run-jnlp-client` is executable? I think elsewhere we do that consistently. Maybe you can use this to also fix the next line - instead of `chmod 777` a simple `chmod +x /tmp/set_java_proxy.sh` should do.